src/org/jitsi/impl/neomedia/rtp/TransportCCEngine.java

```diff
@@ -604,7 +603,7 @@ public RawPacket transform(RawPacket pkt)
                 if (logger.isDebugEnabled())
                 {
                     logger.debug("rtp_seq=" + pkt.getSequenceNumber()
-                            + ",pt=" + pkt.getPayloadType()
+                            + ",pt=" + RawPacket.getPayloadType(pkt)
```

Comment:
Ha! Nice fix. I think we should eventually fix the fact that RawPacket#getPayloadType() and RawPacket#getPayloadType(RawPacket) have different return types...